braindecode/models/eegnet.py

```diff
@@ -89,15 +89,15 @@ def __init__(
         *,
         depthwise_kernel_length: int = 16,
         pool1_kernel_size: int = 4,
-        pool1_stride_size: int = 4,
         pool2_kernel_size: int = 8,
-        pool2_stride_size: int = 8,
         conv_spatial_max_norm: int = 1,
         activation: nn.Module = nn.ELU,
         batch_norm_momentum: float = 0.01,
         batch_norm_affine: bool = True,
         batch_norm_eps: float = 1e-3,
         drop_prob: float = 0.25,
+        final_layer_linear: bool = False,
```

Comment:
Should we use the "correct" EEGNet implementation as default instead? 
Or maybe a deprecation warning in case `final_layer_linear=False` to say that this is not the correct implementation and that the default value will change in the future